persistent-datastores/manifests/postgres-statefulset.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgres
+  labels:
+    app: pg-provider
+  annotations:
+    kots.io/when: '{{repl ConfigOptionEquals "postgres_type" "embedded_postgres" }}'
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: pg-provider
+  serviceName: postgres
+  template:
+    metadata:
+      labels:
+        app: pg-provider
+    spec:
+      containers:
+      - env:
+        - name: PGDATA
+          value: /var/lib/postgresql/data/pgdata
+        # create a db called "postgres"
+        - name: POSTGRES_DB
+          value: postgres
+        # create admin user with name "postgres"
+        - name: POSTGRES_USER
+          value: postgres
```

Comment:
Should these be parameterized? `{{repl ConfigOption "external_postgres_user" }}`

Reply:
this is the internal postgres so they could be, but my thought is, if its internal postgres then the user shouldn't have to configure anything